campaignion_webform_widget/js/widget.js

```diff
@@ -56,11 +56,12 @@ window.addEventListener("DOMContentLoaded", function() {
   messageParent(false);
 });
 
+// Update iframe size after AJAX requests.
```

Comment:
I find the comment a little misleading. Sure, it would be the desired effect, but the function only updates the iframe height where the embedding page can pick it up or ignore…